delta/sdk/src/main/resources/contexts/metadata.json

```diff
@@ -62,6 +62,9 @@
     "Storage": "https://bluebrain.github.io/nexus/vocabulary/Storage",
     "DiskStorage": "https://bluebrain.github.io/nexus/vocabulary/DiskStorage",
     "S3Storage": "https://bluebrain.github.io/nexus/vocabulary/S3Storage",
-    "RemoteDiskStorage": "https://bluebrain.github.io/nexus/vocabulary/RemoteDiskStorage"
+    "RemoteDiskStorage": "https://bluebrain.github.io/nexus/vocabulary/RemoteDiskStorage",
```

Comment:
I think at some point we need to find a way to split the metadata context between plugins and core. Context in `sdk` shouldn't know about types from plugins.

Reply:
Yes. That can be done having in the pluginDef some extraMetadata whatever that gets merged with the original metadata. Let's do that later though

Reply:
Yes, that's something for later.